.mjmlconfig

```diff
@@ -1,5 +1,5 @@
 {
   "packages": [
-   "/Users/iryusa/Workspace/test/lib/MjColumnImageText.js"
+    "/home/meri/git/mjml-component-boilerplate/lib/MjColumnImageText.js"
```

Comment:
There's a bug in MJML core right now, it only accepts absolute path for components but it's fixed in `3.0`